jdaviz/configs/specviz/plugins/line_analysis/tests/test_line_analysis.py

```diff
@@ -80,3 +81,16 @@ def test_line_identify(specviz_helper, spectrum1d):
     la_plugin.selected_line = rest_names[1]
     assert ll_plugin.list_contents['Test List']['lines'][0].get('identify') is True
     assert ll_plugin.list_contents['Test List']['lines'][1].get('identify') is False
+
+
+def test_coerce_unit():
+    q_input = 1 * u.Unit('1E-20 erg m / (Angstrom cm**2 s)')
+    q_input.uncertainty = 0.1 * u.Unit('1E-20 erg m / (Angstrom cm**2 s)')
+    q_coerced = _coerce_unit(q_input)
+    assert q_coerced.unit == u.Unit('erg / (cm**2 s)')
```

Comment:
What do you think?
```python
>>> q_input.unit.decompose(bases=[u.erg, u.cm, u.s])
Unit("1e-10 erg / (cm2 s)")
>>> q_input.decompose(bases=[u.erg, u.cm, u.s])
<Quantity 1.e-10 erg / (cm2 s)>
```

Reply:
that works for this case, but I think we want to support the _user_ bases (except for length, which will cancel once they become the same).

Reply:
OK should we just merge then?

Reply:
Unless you think using decompose with the bases/powers logic I have would be cleaner than recompiling to a string?

Reply:
Maybe not... as I have no idea what weird units will come with arbitrary data. Maybe we just ship this and revisit as needed.